src/HIE/Bios/Cradle.hs

```diff
@@ -373,7 +373,7 @@ stackAction work_dir l fp = do
   -- Same wrapper works as with cabal
   wrapper_fp <- getCabalWrapperTool
   (ex1, _stdo, stde, args) <-
-      readProcessWithExitCodeInDirectory l work_dir "stack" ["repl", "--no-load", "--with-ghc", wrapper_fp, fp ]
+      readProcessWithExitCodeInDirectory l work_dir "stack" ["repl", "--no-nix-pure", "--no-load", "--with-ghc", wrapper_fp, fp ]
```

Comment:
`--no-nix-pure` has no impact when nix is not enabled right?
Maybe add a comment that it does not?